api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/swagger/api/ApiTransformationConfig.java

```diff
@@ -43,7 +44,14 @@ public class ApiTransformationConfig {
     @Bean
     @Scope(value = "prototype")
     public AbstractApiDocService<?, ?> abstractApiDocService(String content) {
-        ObjectMapper mapper = new ObjectMapper(new YAMLFactory())
+        JsonFactory jsonFactory;
+        if (Optional.ofNullable(content).map(String::trim).map(c -> c.startsWith("{")).orElse(false)) {
```

Comment:
Is content really nullable? It doesn't seem like it was before. Also I think the optional is way too much syntax for what it does

```suggestion
        if (String.valueOf(content).trim().startsWith("{"})) {
```

Reply:
String.valueOf(null) retruns "null" which I don't think we want here